data_steward/test/unit_test/achilles_test.py

```diff
@@ -90,7 +92,7 @@ def test_load_analyses(self):
 
     def test_get_run_analysis_commands(self):
         cmd_iter = achilles._get_run_analysis_commands(FAKE_HPO_ID)
-        commands = list(cmd_iter)
+        commands = [achilles.convert_insert_to_append(command)[0] for command in cmd_iter]
```

Comment:
grabbing `[0]` seems unnecessary to me given the test you've written, am I missing something?

Reply:
You are correct, I need to remove it. I had initially set up `achilles.convert_insert_to_append()` to return two outputs, one for the `select` command and the other for the `table_id`, but later split them into two separate functions.